portality/bll/services/concurrency_prevention.py

```diff
@@ -3,7 +3,7 @@
 from portality.ui.messages import Messages
 import redis
 
-class UpdateRequestConcurrencyPreventionService:
+class updateRequestConcurrencyPreventionService:
```

Comment:
Class and method naming conventions are different:

1. Classes - TitleCase
2. Methods: camelCase

The point being to easily distinguish between class and method names and the things they represent.

Therefore, this class name change isn't correct, it was correct before.  It was the method call to obtain the service which was incorrectly cased.